src/Proj4.jl

```diff
@@ -136,6 +142,9 @@ function transform!(src::Projection, dest::Projection, position::Array{Float64,2
     position
 end
 
+transform!(src::Projection, dest::Projection, position::Vector{Float64}; radians::Bool=false) =
+    transform!(src, dest, reshape(position,(1,length(position))), radians=radians)
+
```

Comment:
From what I've seen so far, usually (vector) geographical data doesn't come as nice contiguous matrices, and are more commonly represented as (nested) vector-of-vectors